src/apis/staked-relayer.ts

```diff
@@ -34,6 +35,22 @@ export class DefaultStakedRelayerAPI implements StakedRelayerAPI {
         return activeStakedRelayersMap.map((v) => v[1]);
     }
 
+    async map(): Promise<Map<AccountId, ActiveStakedRelayer>> {
+        const activeStakedRelayers = await this.api.query.stakedRelayers.activeStakedRelayers.entries();
+        const activeStakedRelayerPairs: [
```

Comment:
Is typescript not able to infer types here? Maybe adding `as AccountId` to the `createType` would help?